pkg/backends/jmeter/resources.go

```diff
@@ -204,8 +204,8 @@ func (c *JMeter) NewPod(i int, configMap *coreV1.ConfigMap, podAnnotations map[s
 					},
 					Resources: coreV1.ResourceRequirements{
 						Requests: map[coreV1.ResourceName]resource.Quantity{
-							coreV1.ResourceMemory: resource.MustParse("4Gi"),
-							coreV1.ResourceCPU:    resource.MustParse("1000m"),
+							coreV1.ResourceMemory: resource.MustParse("500Mi"),
```

Comment:
but on real load tests we need more resources, how will it work?

Reply:
This only adjusts resource requests; if needed it can still get more resources up until the resource limits.